docs/architecture/adt-014-proportional-slashing.md

```diff
@@ -0,0 +1,87 @@
+
+```
+slash_amount = (power_1 + power_2 + ... + power_n) // where power_i is the voting power of the ith validator faulting in the period
+```
+
+Now, if someone splits a validator of 10% into two validators of 5% each which both fault, then they both fault in the same window, they both will still get slashed at the sum 10% amount.
+
+However, an operator might still choose to split up their stake across multiple accounts with hopes that if any of them fault independently, they will not get slashed at the full amount.  In the case that the validators do fault together, they will get slashed the same amount as if they were one entity.  There is no con to splitting up.  However, if operators are going to split up their stake without actually decorrelating their setups, this also causes a negative externality to the network as it fills up validator slots that could have gone to others or increases the commit size.  In order to disincentivize this, we want it to be the case such that splitting up a validator into multiple validators and they fault together is punished more heavily that keeping it as a single validator that faults.
+
+We can achieve this by not only taking into account the sum of the percentages of the validators that faulted, but also the *number* of validators that faulted in the window.  One general form for an equation that fits this desired property looks like this:
+
+```
+slash_amount = (sqrt(power_1) + sqrt(power_2) + ... + sqrt(power_n))^2
+```
+
+So now, for example, if one validator of 10% faults, it gets a 10% slash, while if two validators of 5% each fault together, they both get a 20% slash ((sqrt(0.05)+sqrt(0.05))^2).
+
+One will note, that this model doesn't differentiate between multiple validators run by the same operators vs validators run by different operators.  This can be seen as an additional benefit in fact.  It incentivizes validators to differentiate their setups from other validators, to avoid having correlated faults with them or else they risk a higher slash.  So for example, operators should avoid using the same popular cloud hosting platforms or using the same Staking as a Service providers.  This will lead to a more resilient and decentralized network.
+
+
+We can allow some parameterization by multiplying the slash by an on-chain governable parameter.
+
+```
+slash_amount = k * (sqrt(power_1) + sqrt(power_2) + ... + sqrt(power_n))^2  // where k is an on-chain parameter for this specific slash type
+```
+
+This can be used to weight different types of slashes.  For example, we may want to punish liveness faults 10% as severely as double signs.  The k factor can also be something other than a constant, there is some research on using things like inverse gini coefficients to mitigate some griefing attacks, but this an area for future research.
+
+There can also be minimum and maximums put in place in order to bound the size of the slash percent.
+
+### Implementation
```

Comment:
F1 already tracks slash events. Ideally we should combine these trackers.

Reply:
Isn't this a conflation of concerns?  Don't want to add an unnecessary dependency between `distr` and `slashing`.

Reply:
Hmm, I suppose not, storing them twice isn't great though. Maybe there should be a third module just to track slash events, imported by both?

Reply:
@alexanderbez Is this what the evidence module will do?

Reply:
Evidence module does not track slashing events nor did we plan on it doing so -- I'm not sure it makes much sense.

Why not have the slashing module track slash events? I'm sure we can think of a clean way to retrofit that into distribution (e.g. via a minimal interface).

Reply:
Sure, I think it makes sense to track in slashing and refactoring `distribution` to be dependent on `slashing`.  I think that's out of scope for this ADR though.  A separate issue can be opened to make `distribution` use the slash events tracker in `slashing`.